deploy/helm-chart/templates/rbac.yaml

```diff
@@ -37,6 +37,7 @@ rules:
 {{- end }}
   verbs:
   - list
+  - get
```

Comment:
Is this change for secrets only?

Reply:
This is needed to get the kube-system namespace for both telemetry and usage.

It's a lesser privileged option than `list`, so I don't think we need to conditionalize this.